v1/lib/gamefield.php

```diff
@@ -1,26 +1,51 @@
 <?php
+/**
+ * @file
+ * @version 0.1
+ * @copyright 2016 CN-Consult GmbH
+ * @author Max Späth <max.spaeth@cn-consult.eu>
+ */
+
+require_once "cell.php";
 
 /**
- * Created by PhpStorm.
- * User: Max
- * Date: 15.01.2016
- * Time: 12:02
+ * This class is the gamefield which contains an array of cells.
  */
 class GameField
 {
     /** @var  Cell[] */
     private $cells;
+    private $width;
+    private $height;
 
     /**
      * GameField constructor.
-     * @param $_cells cell[] Array which contains instances of cell.
+     *
+     * @param int $_width Width of the gamefield (x-axis)
+     * @param int $_height Height of the gamefield (y-axis)
      */
-    public function __construct($_cells)
+    public function __construct($_width, $_height)
     {
-        $this->cells = $_cells;
+        $this->width = $_width;
+        $this->height = $_height;
+        $this->cells = array();
+        $this->createCells();
     }
 
     /**
+     * Creates the array of cells which is the gamefield.
```

Comment:
remove "which is the gamefield", because we know already, that we are in gamefield context.